test/acceptor_SUITE.erl

```diff
@@ -1193,6 +1197,54 @@ supervisor_unexpected_message(_) ->
 	{'EXIT', _} = begin catch ranch:get_port(Name) end,
 	ok.
 
+supervisor_acceptors_conns_sups_10to1(_) ->
+	ok = do_supervisor_acceptors_conns_sups(10, 1).
```

Comment:
You need to document the test by calling the doc("blabla") function at the start like in other test cases. Say what it does.